Duplicati/Library/AutoUpdater/UpdaterManager.cs

```diff
@@ -1062,6 +1062,36 @@ private static int RunMethod(System.Reflection.MethodInfo method, string[] args)
             return best;
         }
 
+        private static void CleanOlderUpdates()
+        {
+            // This function is only called under an update environment, therefore SelfVersion will always be the best version
+            KeyValuePair<string, UpdateInfo> best = new KeyValuePair<string, UpdateInfo>(AppDomain.CurrentDomain.SetupInformation.ApplicationBase, SelfVersion);

```

Comment:
Does this need to be a `KeyValuePair`?  It looks like we only access the `Value` part of it.

Reply:
I agree, will change it tomorrow at work

Reply:
Done